src/core_modules/capture-core/components/DataEntries/SingleEventRegistrationEntry/DataEntryWrapper/DataEntry/epics/getConvertedNewSingleEvent.js

```diff
@@ -35,7 +36,6 @@ export const getNewEventServerData = (state: ReduxState, formFoundation: RenderF
             program: state.currentSelections.programId,
             programStage: formFoundation.id,
             orgUnit: state.currentSelections.orgUnitId,
-            ...getApiCategoriesArgument(state.currentSelections.categories),
```

Comment:
Doesn't seem to work when adding events for event programs. This might be the culprit?